src/main/java/frc/robot/Constants.java

```diff
@@ -5,11 +5,28 @@
 package frc.robot;
 
 /**
- * The Constants class provides a convenient place for teams to hold robot-wide numerical or boolean
- * constants. This class should not be used for any other purpose. All constants should be declared
- * globally (i.e. public static). Do not put anything functional in this class.
+ * The Constants class provides a convenient place for teams to hold robot-wide
+ * numerical or boolean constants. This class should not be used for any other
+ * purpose. All constants should be declared globally (i.e. public static). Do
+ * not put anything functional in this class.
  *
- * <p>It is advised to statically import this class (or one of its inner classes) wherever the
- * constants are needed, to reduce verbosity.
+ * <p>
+ * It is advised to statically import this class (or one of its inner classes)
+ * wherever the constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+    public static final class TankConstants {
```

Comment:
What is the difference between the values in this file and the ones in config.txt? We should avoid having duplicate information.

Reply:
The original plan was to move everything from config.txt to Java. We wanted to leave in the functionality in case we're using text-based configuration in the future, though, and I think we just accidentally didn't clear the text file itself.